packages/cloud-cognitive/src/components/Datagrid/styles/_useStickyColumn.scss

```diff
@@ -23,25 +23,41 @@
   right: 0;
 }
 
-.#{$block-class}__left-sticky-column-cell {
+.#{$block-class}__left-sticky-column-header {
   /* stylelint-disable-next-line declaration-no-important */
   position: sticky !important;
+  z-index: 1;
   left: 0;
-  display: flex;
-  align-items: center;
-  border-right: 1px solid $ui-03;
 }
```

Comment:
Looks like this change actually affects the left sticky columns when there is no index column to the left too — right border is now missing but should also use `$active-light-ui`.

![image](https://user-images.githubusercontent.com/1717398/220247889-b289f825-280c-4323-95b6-0fd37fe7e831.png)

Reply:
If we are reverting the row index changes, I hope the border will restore. Otherwise, I'll fix it.